compose/utils.py

```diff
@@ -151,3 +152,21 @@ def unquote_path(s):
     if s[0] == '"' and s[-1] == '"':
         return s[1:-1]
     return s
+
+
+def generate_random_id():
```

Comment:
Why do we use such a complicated algorithm to generate a random identifier? And is there any specific reason we don't want a fully numerical ID?

Otherwise `uuid.uuid4().hex` sounds good to generate a hexadecimal identifier, no?

Reply:
The code is a transposition of https://github.com/moby/moby/blob/master/pkg/stringid/stringid.go ; we could go with a UUID instead.